TECHNICAL.md

```diff
@@ -121,10 +123,10 @@ begin packages
   chd6 GWF_1/chd.chd
   npf6 GWF_1/npf.npf
   ic6 GWF_1/ic.ic
-  wel6 GWF_1/wel.wel wel1
+  wel6 GWF_1/wel.wel WELLS_MSW
   wel6 GWF_1/wel2.wel wel2
   sto6 GWF_1/sto.sto
   oc6 GWF_1/oc.oc
 end packages
 ```
-The optional argument values "wel1" and "wel2", specify the packagenames to be printed in the water budget .lst file.
+
```

Comment:
I though this line of information was quite useful, so that people can look up their water budgets in the .lst file.